src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1732,6 +1732,73 @@
     /**
      * A left parenthesis ({@code (}).
      *
+     * <p>For example:</p>
+     * <pre>
+     *   String str = new String();
+     *   for (int i = 0; i &lt; 2; i++) {
+     * 	        str += (i*((5 + 7) - 3));
+     *   }
```

Comment:
lets make it more compact to focus on `(` as much as possible:
```
        Integer val = new Integer();
        while (false) {
            val += (-3);
        }
```

Reply:
@romani Thanks for review and advice. I updated this, but two tests failed.